src/item/ItemTypeIds.php

```diff
@@ -324,8 +324,9 @@ private function __construct(){
 	public const SPIRE_ARMOR_TRIM_SMITHING_TEMPLATE = 20285;
 	public const PITCHER_POD = 20286;
 	public const NAME_TAG = 20287;
+	public const MACE = 20289;
 
-	public const FIRST_UNUSED_ITEM_ID = 20288;
+	public const FIRST_UNUSED_ITEM_ID = 20290;
```

Comment:
This file should be autogenerated using the composer script included with pocketmine's build tools.